framework/src/iotbus/iotbus_uart.c

```diff
@@ -166,6 +184,7 @@ iotbus_uart_context_h iotbus_uart_init(const char *path)
 	}
 
 	handle->fd = fd;
+	handle->device = _iotbus_get_dev_number(path);
```

Comment:
Could you let me know what is the meaning of device number, and who and how to manage it?
It might be dangerous if the input path is not correctly assigned because of while loop in line 147 above.

Reply:
@SeonghoByeon 
In iotbus_uart, it always use the path, '/dev/ttyS*' to open fd.
If the path is wrong, the logic stops at lin 170 before calling '_iotbus_get_dev_number'